icd/CMakeLists.txt

```diff
@@ -80,9 +80,14 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
 
     if(CMAKE_BUILD_TYPE_RELEASE)
         target_compile_options(xgl PRIVATE -O3)
-        execute_process(COMMAND ${CMAKE_C_COMPILER} -dumpversion OUTPUT_VARIABLE GCC_VERSION)
-        if(GCC_VERSION VERSION_GREATER 5.3 OR GCC_VERSION VERSION_EQUAL 5.3)
-            add_definitions("-flto -fuse-linker-plugin -Wno-odr")
+        if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+            execute_process(COMMAND ${CMAKE_C_COMPILER} -dumpversion OUTPUT_VARIABLE GCC_VERSION)
+            if(GCC_VERSION VERSION_GREATER 5.3 OR GCC_VERSION VERSION_EQUAL 5.3)
+                add_definitions("-flto -fuse-linker-plugin -Wno-odr")
+                message(WARNING "LTO enabled for XGL")
+            endif()
+        elseif(CMAKE_CXX_COMPILER_ID MATCHES "[Cc]lang")
```

Comment:
I think upper case Clang should be sufficient ([list of compiler ids](https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html)), otherwise if we support lowercase clang here, we should also change all the other matches for consintency.

Reply:
ok, changed